src/Compilers/Core/Portable/Operations/OperationInterfaces.xml

```diff
@@ -831,7 +831,7 @@
       </Comments>
     </Property>
   </Node>
-  <Node Name="IUnaryOperation" Base="IOperation">
+  <Node Name="IUnaryOperation" Base="IOperation" VisitorName="VisitUnaryOperator">
```

Comment:
>VisitUnaryOperator [](start = 62, length = 18)

I think we could have a method to figure out the `Visit` method name from the node name. We would strip the `I` and any trailing `Operation`.

Reply:
Talked with Julien offline about this. I'm going to keep this the way it is, so that the info on exceptions to our standard rules is declared in this XML rather than specialized knowledge in the builder.

---
In reply to: [313624646](https://github.com/dotnet/roslyn/pull/37791#discussion_r313624646) [](ancestors = 313624646)